<launch>
   <node name="rmp_test_node" pkg="cpt_ros" type="rmp_test_node" output="screen">

     <param name="zero_angle" value="0.26"/>
    <param name="zero_x" value="0.45"/>
    <param name="zero_y" value="0.9"/>
    <param name="zero_z" value="0.64"/>
    <param name="off_path" value="/home/mpantic/Desktop/rss/surfaces/doublycurved.off" />
         <remap from="/spacenav/joy" to="/joy" />

</node>
 <node name="joy_node" pkg="joy" type="joy_node" output="screen">
    </node>

</launch>
